src/platform/landing-pages/dev-template.ejs

```diff
@@ -35,14 +35,25 @@
     <%= modifyScriptTags(htmlWebpackPlugin.tags.bodyTags) %>
 
     <script nonce="**CSP_NONCE**" type="text/javascript">
+      function scrollToContent(e) {
+        e.preventDefault();
+        e.target.hidden = true;
+        const contentRoot = document.querySelector('#content');
+        const contentH1 = document.querySelector('#content h1');
+        if (contentH1) {
+          window.scrollTo(0, contentH1.offsetTop);
+        } else {
+          window.scrollTo(0, contentRoot.offsetTop);
+        }
+      }
       window.VetsGov = window.VetsGov || {};
       window.VetsGov.headerFooter = <%= JSON.stringify(headerFooterData) %>;
     </script>
 
   </head>
 
   <body class="merger">
-    <a class="show-on-focus" href="#content">Skip to Content</a>
+    <a class="show-on-focus" href="#" onclick="scrollToContent(event)">Skip to Content</a>
```

Comment:
Changing this as `#` will fail our accessibility, valid hrefs have to be a url or anchor. Can we leave this as `#content` what way it will work without JS as well?

Reply:
Good catch! It should still work with `#content` now that I've added `e.preventDefault()`. Thanks!